QuasistaticBrownianThermalNoise/QuasistaticBrownianThermalNoise.cpp

```diff
@@ -333,15 +335,37 @@ CylTransFunc::CylTransFunc(double coatThick, double halfCylThick):
 
     //The negative sign here is because Tzj should point in the -z 
     //direction, so the mirror is compressed, not stretched.
-    myValue = -1.0 * F0 * exp(myValue/(r0*r0));
-
     //Normalize: e.g. Liu+ Eq. (2): if F0=F0, I'd better divide by pi*r0^2
-    myValue /= M_PI * r0 * r0;
+
+    int mWhichProfile = TEM00;
+
+    switch(mWhichProfile) {
+    case(TEM00):
+      myValue = -1.0 * F0 * exp(myValue/(r0*r0));
+      myValue /= M_PI * r0 * r0;
+      break;
+    case(TEM02):
+      myValue = -1.0 * F0 * exp(myValue/(r0*r0)) * (4*p(1)*p(1)/(r0*r0)-2)*(4*p(1)*p(1)/(r0*r0)-2);
+      myValue /= 8 * M_PI * r0 * r0;
```

Comment:
Sorry I missed this before, but be aware that some older versions of python might treat your numbers as integers, not floats, if no decimal points. To be safe, Please put ".0" after all hard-coded floating-point numbers.